docs/content/latest/explore/ysql-language-features/partitions.md

```diff
@@ -109,7 +106,7 @@ The following example shows how to quickly remove old data via dropping the part
 DROP TABLE employees_2019_02;
```

Comment:
Above: "(you need to take an `ACCESS EXCLUSIVE` lock on the partitioned `employees` table)"

this statement makes it seem like a user needs to do something to achieve this DROP statement. The right statement here is "Note that this operation takes an ACCESS EXCLUSIVE lock on the partitioned employees table"

Reply:
Changed.